packages/validator-query/spec/AmpValidatorQuerySpec.js

```diff
@@ -0,0 +1,171 @@
+/**
```

Comment:
Does this run for you if `npm run test:node` is invoked in the root directory? I'm not 100% sure if it's my config but this doesn't seem to actually do anything, perhaps because in `jasmine.json` it looks for spec files that match `packages/*/spec/**/*[sS]pec.js`.

Reply:
Well, it's not obvious from the log, but if I try to make a test break (e.g. change one expected result), `npm test` complains, so I'm quite confident it runs.

The pattern also matches this file if I'm reading it correctly (`**` should also work for `.`)?

Reply:
You're right, comment withdrawn!